services/web/server/src/simcore_service_webserver/projects/_controller/nodes_rest.py

```diff
@@ -331,7 +331,10 @@ async def _stop_dynamic_service_task(
         return web.json_response(status=status.HTTP_204_NO_CONTENT)
 
 
-TaskRegistry.register(_stop_dynamic_service_task)
+def register_stop_dynamic_service_task(app: web.Application) -> None:
+    TaskRegistry.register(
+        _stop_dynamic_service_task, allowed_errors=(web.HTTPNotFound,), app=app
```

Comment:
Question: say I now modify _stop_dynamic_service_task and add another exception (which by the way since this is not a controller should not directly raise http errors). What will I see?